.circleci/config.yml

```diff
@@ -74,17 +72,16 @@ php_build: &php_build
     - run:
         name: Composer Metrics
         command: composer metrics
-  
-  # Generate code coverage artifacts and upload to Codecov.
+        
     - when:
         condition: 
             and:
               - equal: [ "7.4", << parameters.php-version >> ]
               - equal: [ " ", << parameters.dependencies >> ]  
         steps:
+           # Generate code coverage artifacts and upload to Codecov.
           - run: phpdbg -d memory_limit=-1 -qrr vendor/bin/phpunit --coverage-html build/coverage-report --coverage-clover coverage.xml
-          - codecov/upload:
-              file: 'coverage.xml'
+          - run: bash <(curl -s https://codecov.io/bash) 
```

Comment:
@shishir-intelli , This bash approach seems to be deprecated 
https://docs.codecov.com/docs/about-the-codecov-bash-uploader

Reply:
Replaced bash uploader with codecov uploader
PR https://github.com/apigee/apigee-client-php/pull/200